### ISSUE TITLE
[5.0.0-rc1] 'Select preferred Language(s)' causes forgotten Server at app restart and whitescreen until resized

### ISSUE THREAD

**Issue by NeroBurner** [mattermost/desktop#1754]
- [x] This is not a troubleshooting question. [Troubleshooting questions go here: http://www.mattermost.org/troubleshoot/](http://www.mattermost.org/troubleshoot/).
- [x] This doesn't reproduce on web browsers (such as in Chrome). If it does, [issue reports go to the Mattermost Server repository](https://github.com/mattermost/platform/issues).
- [x] I have read [contributing guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md).

---

**Summary**

with [5.0.0-rc1](https://github.com/mattermost/desktop/releases/tag/v5.0.0-rc1) the set servers (or all config) is forgotten when restarting the app

**Environment**
- Operating System: KUbuntu 20.04
- Mattermost Desktop App version: 5.0.0-rc1
- Mattermost Server version: 5.37.1

**Steps to reproduce**
- Have `mattermost-desktop` set up and successfully connected to your server (either with 5.0.0-rc1 or wit previous stable 4.7.2)
- restart `mattermost-desktop`
- be greeted with whitescreen and no server connected
- resize window to get the "connect to server" dialog

**Expected behavior**
remember server over app restarts

**Observed behavior**
forgetting server and having whitescreen

Whitescreen on restart

![whitescreen on restart](https://user-images.githubusercontent.com/9076163/134294019-54cda6fe-857c-4327-ae90-54c6f232eafa.png)

window after resizing showing "Add Server" dialog
![window after resizing](https://user-images.githubusercontent.com/9076163/134294102-0b345c7d-0ba3-493b-9835-28d11709f506.png)

nothing interesting in commandline output
```sh
$ mattermost-desktop 
08:34:23.392 › Current working directory is /home/nero, changing into /opt/Mattermost
08:34:23.702 › config.autostart has been configured: true
08:34:23.703 › config.autostart has been configured: true
08:34:23.786 › preloading with /opt/Mattermost/resources/app.asar/modalPreload.js
08:34:23.812 › config.autostart has been configured: true
```

I also tried to remove the `.config/Mattermost` folder, connected to the server, and restartet the app. Also forgot the server again

**Possible fixes**

**Comment by amyblais** [comment]
@devinbinnie I partially reproduced this. When I downloaded RC-1 on Mac for the first time, all my configured servers disappeared. Is that a known issue?

**Comment by devinbinnie** [comment]
@amyblais The only known issue is you might run into issues upgrading from one of the nightly 5.0 builds to 5.0-rc1. If you're coming from v4.7, that should be working correctly.

@NeroBurner Does this reliably reproduce for you if you upgrade from v4.7.2 to v5.0-rc1?

**Comment by NeroBurner** [comment]
I can't reproduce it anymore. As of writing this Issue the server was forgotten by upgrading from v4.7.2 to v5.0-rc1 and even when starting with a fresh `~/.config/Mattermost` folder with `v5.0-rc1`

now it just works. Closing this issue. Will reopen if I can reproduce it again

**Comment by NeroBurner** [comment]
I can reproduce:
- open settings
- set the field `Select preferred Language(s)` to contain `German`
  ![Screenshot_20210927_101012](https://user-images.githubusercontent.com/9076163/134869591-d6b955ce-5a97-4141-bffd-89e1a86e57e7.png)
- restart Mattermost
- get blank screen

**Comment by NeroBurner** [comment]
can confirm fixed with `mattermost-desktop-5.0.0-rc2`
